packages/core/src/components/table/table-body-row/table-body-row.tsx

```diff
@@ -61,6 +64,18 @@ export class TdsTableBodyRow {
     selectedRows: any[];
   }>;
 
+  /** Event emitted when a row is clicked. */
+  @Event({
+    eventName: 'tdsClick',
+    composed: true,
+    cancelable: false,
+    bubbles: true,
+  })
+  tdsClick: EventEmitter<{
+    tableId: string;
+    rowIndex: number;
```

Comment:
might wanna return the click event here aswell.

Reply:
Thanks! 👍